util/test_timeline.py

```diff
@@ -312,11 +312,18 @@ def check_event(event, timelist, state):
 
             # Check the timeline drift for anomolous timings
             drift = entry["time"] - timeline_position
-            print(
-                "{:.3f}: Matched entry: {} {} ({:+.3f}s)".format(
-                    timeline_position, entry["time"], entry["label"], drift
-                )
+            entry_text = "{:.3f}: Matched entry: {} {} ({:+.3f}s)".format(
+                timeline_position, entry["time"], entry["label"], drift
             )
+            if abs(drift) > 1:
+                print("\033[91m" + entry_text + "\033[0m")
```

Comment:
Can I suggest `print_warning` or `print_error` helper functions for these? Or something along these lines?

Reply:
That seems reasonable, yes.